src/main/java/com/github/kokorin/jaffree/process/ProcessHandler.java

```diff
@@ -180,7 +193,7 @@ protected T interactWithProcess(final Process process) {
             status = process.waitFor();
             LOGGER.info("Process has finished with status: {}", status);
 
-            waitForExecutorToStop(executor, EXECUTOR_TIMEOUT_MILLIS);
+            waitForExecutorToStop(executor, executorTimeoutMillis);
```

Comment:
I think it makes sense to allow a user to disable timeout by setting 0 (zero) value.

Reply:
OK - I will detect that and short-circuit, else you would get a log line like:
```
WARN: Executor hasn't stopped in 0 millis, won't wait longer
```